tools/CheckCommits.sh

```diff
@@ -10,7 +10,40 @@ IFS=$'\n'
 # Get the last 300 commits and check those for bad messages. TravisCI doesn't
 # allow us to get develop (at least not easily) so we can't just find and check
 # the added commits.
-COMMIT_LINES=`git log -n 300 --oneline`
```

Comment:
Okay, updated, since it's just a single comment change I rebased right away. Please let us know when you think it's good to merge :)